ci-operator/config/openstack-k8s-operators/test-operator/openstack-k8s-operators-test-operator-18.0.0-proposed.yaml

```diff
@@ -0,0 +1,93 @@
+base_images:
+  cli:
+    name: "4.16"
+    namespace: ocp
+    tag: cli
+  openstack-operator-ci-tempest:
+    name: tools
+    namespace: openstack-k8s-operators
+    tag: openstack-operator-tempest-container
+  opm-builder:
+    name: opm-builder
+    namespace: ci
+    tag: latest
+build_root:
+  from_repository: true
+images:
+- dockerfile_literal: |
+    FROM src
+    RUN pip3 install python-openstackclient
+    COPY oc /usr/bin/oc
+    COPY opm /usr/bin/opm
+  from: src
+  inputs:
+    cli:
+      paths:
+      - destination_dir: .
+        source_path: /usr/bin/oc
+    opm-builder:
+      paths:
+      - destination_dir: .
+        source_path: /bin/opm
+  to: openstack-operator-ci-build-deploy
+- dockerfile_literal: |
+    FROM src
+    RUN pip3 install pre-commit
+  from: src
+  to: openstack-operator-ci-pre-commit
+- build_args:
```

Comment:
Only if you plan to add kuttl tests and a kuttl job, otherwise you can remove L38 to L49

Reply:
will remove it to start without content not being used